app/Utils/ArrayFunctions.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Utils;
+
+function is_conditional_array(array $array): bool
```

Comment:
Lepiej by było te funkcje dać jako statyczne publiczne w klasie np. ConditionalArrayRule, np

ConditionalArrayRule::isConditionalRule

ConditionalArrayRule::processCondition

Te nazwy może się zrobią trochę przydługie, ale no, namespace'y lepiej działają z klasami niż z funkcjami

Reply:
Zmieniłem to. Musiałem jeszcze poprawić jakieś test case'y.